Drive_2_0_fieldCentric.java

```diff
@@ -361,6 +365,18 @@ else if (isArmCmdExtend) {
         }
         lastTimeGrabberToggledMs = (int) currentTime.milliseconds();  // refresh 'lastTime', regardless of which direction we went
       }
+
+      // drop bottom pixel code
+      boolean isBottomPixelDebounceTimeElapse = ((int) currentTime.milliseconds() - lastTimeBottomPixelMs)
+                                                                                > GRABBER_BOTTOM_PIXEL_DEBOUNCE_MS;
+
+      if (driverCmd_DropBottomPixel ) {
+        flipper.setPosition(CLAW_FLIP_SERVO_NORMAL_POS);
+        grabber.setPosition(GRABBER_SERVO_OPEN_A_LITTE_POS);
+        sleep(100);
+        grabber.setPosition(GRABBER_SERVO_OPENED_POS);
+        lastTimeBottomPIxelMs = currentTime.milliseconds();
+      }
```

Comment:
@gwarby  this code is calculating whether debounce time has elapsed, and updating the corresponding lastTime variable, but I don't see where it is actually using `driverCmd_DropBottomPixel` in the logic.